controllers/release/adapter.go

```diff
@@ -717,6 +1030,25 @@ func (a *adapter) getEmptyReleaseServiceConfig(namespace string) *v1alpha1.Relea
 	return releaseServiceConfig
 }
 
+// registerTenantCollectorsProcessingData adds all the Release Tenant Collectors processing information to its Status
+// and marks it as tenant collectors processing.
+func (a *adapter) registerTenantCollectorsProcessingData(releasePipelineRun *tektonv1.PipelineRun) error {
+	if releasePipelineRun == nil {
+		return nil
+	}
+
+	patch := client.MergeFrom(a.release.DeepCopy())
+
+	a.release.Status.CollectorsProcessing.TenantCollectorsProcessing.PipelineRun = fmt.Sprintf("%s%c%s",
+		releasePipelineRun.Namespace, types.Separator, releasePipelineRun.Name)
+
+	a.release.MarkTenantCollectorsPipelineProcessing()
+
+	e := a.client.Status().Patch(a.ctx, a.release, patch)
```

Comment:
why define `e` just to return it? Just return the function call like every other function like this does

Reply:
Fixed.